src/helpers/heart-rate-data-providers/intraday-aggregates.tsx

```diff
@@ -0,0 +1,29 @@
+import MyDataHelps, { Guid, DeviceDataV2AggregateQuery, DeviceDataV2AggregatePage, DeviceDataV2Aggregate } from "@careevolution/mydatahelps-js";
+
+export default async function (query: DeviceDataV2AggregateQuery): Promise<DeviceDataV2Aggregate[]> {
```

Comment:
What does this have to do with heart rate data - this seems like a general "queryAllDeviceDataV2" function more than anything to do with heart rates?

Reply:
So seems like it would go in the same bucket as query all survey answers etc.